ui/i18n/messages.json

```diff
@@ -242,6 +244,7 @@
   "operationButton.bulk.poll.error": "Error while executing {0}",
   "operationButton.bulk.poll.running": "{0} is running ({1}/{2})",
   "operationButton.bulk.poll.scheduled": "{0} is scheduled. You will be notified when it starts.",
+  "parentInspectorButton.tooltip": "Parent Inspector",
```

Comment:
It this label is not used in this repository but only in the nuxeo web ui repository, then that's where it should live.